src/kagglehub/auth.py

```diff
@@ -85,7 +85,7 @@ def _notebook_login(validate_credentials) -> None:
     )
     display(login_token_widget)
 
-    def on_click_login_button(t):  # noqa: ARG001
+    def on_click_login_button(t: str) -> None:  # noqa: ARG001
```

Comment:
I think you can use "_: str" to denote unused argument and remove the need for the #noqa annotation.